pac4j-saml/src/main/java/org/pac4j/saml/sso/impl/SAML2AuthnResponseValidator.java

```diff
@@ -212,8 +215,21 @@ protected void validateSamlProtocolResponse(final Response response, final SAML2
 
         validateSuccess(response.getStatus());
 
-        if (wantsResponseSigned && response.getSignature() == null) {
-            throw new SAMLSignatureValidationException("Signature not found");
+        if (wantsResponsesSigned && response.getSignature() == null) {
+            if (logger.isDebugEnabled()) {
+
+                try {
+                    logger.debug(
+                            "Unable to find a signature on the SAML response returned. Pac4j is configured to enforce "
+                            + "signatures on SAML2 responses from identity providers and the returned response\n{}\n"
+                            + "does not contain any signature",
+                            SerializeSupport.prettyPrintXML(XMLObjectSupport.marshall(response)));
```

Comment:
You should 

1- Use this https://github.com/pac4j/pac4j/blob/master/pac4j-saml/src/main/java/org/pac4j/saml/util/Configuration.java#L97

or 

2- Change that method to use your solution above instead.